extensions/wikia/ListGlobalUsers/ListGlobalUsers.setup.php

```diff
@@ -5,7 +5,7 @@
 $GLOBALS['wgAutoloadClasses']['DatabaseGlobalUsersService'] = __DIR__ . '/DatabaseGlobalUsersService.php';
 
 $GLOBALS['wgAutoloadClasses']['SpecialListStaff'] = __DIR__ . '/alias/SpecialListStaff.php';
-$GLOBALS['wgAutoloadClasses']['SpecialListVstf'] = __DIR__ . '/alias/SpecialListVstf.php';
+$GLOBALS['wgAutoloadClasses']['SpecialListSoap'] = __DIR__ . '/alias/SpecialListSoap.php';
```

Comment:
As in ucp, should we have this already with V1 version?

Reply:
I would keep it here since it's the one that removes VSTF group and switch everything what's left to SOAP